src/components/MainContainer/ContainerDesktop.tsx

```diff
@@ -0,0 +1,35 @@
+'use client'
+
+import { HTMLAttributes } from 'react'
+import { useAlertContext } from '@/app/providers'
+import { FooterDesktop } from '@/components/Footer'
+import { TopPageHeader } from '@/shared/walletConnection/components/topPageHeader/TopPageHeader'
+import { SidebarDesktop } from './sidebars/SidebarDesktop'
+import { Alert } from '../Alert'
+import { HeaderDesktop } from './headers/HeaderDesktop'
+import { cn } from '@/lib/utils'
+
+export function ContainerDesktop({ children, className, ...props }: HTMLAttributes<HTMLDivElement>) {
+  const { message, setMessage } = useAlertContext()
+  return (
+    <div {...props} className={cn('flex min-h-screen max-w-6xl mx-auto', className)}>
```

Comment:
The class here max-w-6xl is shrinking the dApp. It should take full screen

Reply:
we need to discuss it with the designers. I fixed the max width deliberately because in my opinion the width should not increase limitlessly. Look at any modern web site, say Ethereum.com. Every site has a max width limit. I think we should also have it

Reply:
Could we go with the fullscreen pending when we have the conversation with design so this PR can be merged? For me everything else is good to go since there'd be a PR to fix tackle the colours.

Reply:
I discussed with Vlad and he gave me the concrete max width value which I applied here